src/routes/welcome/start/index.tsx

```diff
@@ -1,49 +1,81 @@
 import { ButtonV2, Spacer, Text } from "@arconnect/components";
 import { ArrowRightIcon } from "@iconicicons/react";
-import { useLocation, useRoute } from "wouter";
 import { motion } from "framer-motion";
-import { useMemo } from "react";
-import Screenshots from "~components/welcome/Screenshots";
 import browser from "webextension-polyfill";
 import styled from "styled-components";
-import Ecosystem from "./ecosystem";
-import Arweave from "./arweave";
+import { useLocation } from "~wallets/router/router.utils";
+import type { CommonRouteProps } from "~wallets/router/router.types";
 
-export default function Start() {
-  // router
-  const [, setLocation] = useLocation();
+import { ArweaveWelcomeView } from "./arweave";
+import { EcosystemWelcomeView } from "./ecosystem";
+import { ScreenshotsWelcomeView } from "./screenshots";
+import { Redirect } from "~wallets/router/components/redirect/Redirect";
 
-  // route
-  const [, params] = useRoute<{ page: string }>("/start/:page");
+interface PageInfo {
+  // i18n key
+  title: string;
+  // i18n key
+  content: string;
+  // arwiki link
+  arWiki?: string;
+}
 
-  // page of the setup
-  const page = useMemo(() => {
-    const page = Number(params?.page || "1");
+const pagesInfo: PageInfo[] = [
+  {
+    title: "what_is_arweave",
+    content: "about_arweave",
+    arWiki: "https://arwiki.wiki/#/en/Arweave"
+  },
+  {
+    title: "what_is_the_permaweb",
+    content: "about_permaweb",
+    arWiki: "https://arwiki.wiki/#/en/the-permaweb"
+  },
+  {
+    title: "what_is_arconnect",
+    content: "about_arconnect"
+  }
+] as const;
 
-    if (![1, 2, 3].includes(page)) return 1;
+const Views = [
```

Comment:
This could/should be implemented using `<Routes>`. See https://github.com/arconnectio/ArConnect/issues/567.